### PR TITLE
Wire up Travis config

### PR DESCRIPTION
**Pull request by jayzes**
Just need to add the Danger token to the Travis vars.

### CODE REVIEW

**Comment by ellodanger** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Please provide a linked tracker story for this PR in the description</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">danger</a>
</p>

File: package.json [hunk 1]
```diff
@@ -112,9 +114,8 @@
   },
   "devDependencies": {
     "babel-eslint": "7.2.2",
-    "babel-jest": "19.0.0",
-    "babel-preset-flow": "6.23.0",
-    "babel-preset-react": "6.24.1",
+    "babel-jest": "^19.0.0",
+    "babel-preset-react-native": "^1.9.1",
```
Comment (rynbyjn) [inline package.json]: Apparently our lock down package version danger rule is still busted..

Comment (jayzes) [inline package.json]: Son of a...

Comment (jayzes) [inline package.json]: Ah, right - these are `devDependencies` and the rule doesn't check for that.

**Comment by jayzes** [comment]
@steam @rynbyjn tried for a few hours to get the gradle tests going and had minimal luck so I'm going to give up on that one for now. This should help by getting JS specs in, at the very least.

**Comment by jayzes** [comment]
Left the attempt commit and it's revert in the history for posterity/reference.

**Pull request merged**